### ISSUE TITLE
Incomplete/incorrect instructions in readme 

### ISSUE THREAD

**Issue by jssee** [ido-nvim/files#7]
This looks like an interesting family of plugins I've been trying to give a shot but it seems just following the instructions on the readme won't allow it. I'm currently using packer to handle packages, this is the `Ido` block

```
  use {
    "ido-nvim/files",
    "ido-nvim/core",
    config = function ()
      local ido = require"ido"

      ido.setup{
        packages = {
          files = {}
        }
      }
    end
  }
```

identical to the Readme instructions. Then I run:
`:lua require("ido").module.run("search")`

and get the following error:
`main() not present or non existant module: search`

same error happens no matter what module I input. This leads me to believe I'm missing something, but since the instructions are so sparse I can't seem to figure out what. Looking through the source and I see that there's been some recent API changes, and the module loading in particular seems to have change a fair bit, maybe something was lost in translation?

**Comment by shoumodip** [comment]
Check it out now. You are right, it was a slight change in the API I forgot to update in the README. Sorry about that!

TL;DR use this:
```vim
:lua require("ido").module.run("files/search")
```

**Comment by jssee** [comment]
This fixes it, thanks.
